src/components/DonutChart.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { omit } from 'lodash';
+import { Subtract } from 'utility-types';
+import { VisualizationObject } from '@gooddata/typings';
+
+import { DonutChart as AfmDonutChart } from './afm/DonutChart';
+import { ICommonChartProps } from './core/base/BaseChart';
+import { convertBucketsToAFM } from '../helpers/conversion';
+
+export interface IDonutChartBucketProps {
+    measures: VisualizationObject.BucketItem[];
+    viewBy?: VisualizationObject.IVisualizationAttribute;
+    filters?: VisualizationObject.VisualizationObjectFilter[];
+}
+
+export interface IDonutChartProps extends ICommonChartProps, IDonutChartBucketProps {
+    projectId: string;
+}
+
+type IDonutChartNonBucketProps = Subtract<IDonutChartProps, IDonutChartBucketProps>;
+
+/**
+ * [DonutChart](http://sdk.gooddata.com/gdc-ui-sdk-doc/docs/next/pie_chart_component.html)
```

Comment:
Don't forget update this link once Donut chart documentation is ready

Reply:
Noted.
Thank Ivan.